### DISCUSSION TITLE
FrontEnd - Change Color last login cell in user panel with a specific condition

### DISCUSSION THREAD

**Discussion by TecnicoDelphi** [kimai/kimai#3227] [Questions & Answers] [answered]
Hello everyone, I'm a junior so be patient. :-P

The Boss asked me to change the color of the lastlogin text on the user page when the user has not logged in for more than 24h.

in: templates/user/index.html.twig i made the following change:

/* 86400 seconds for 24hh */
{% if entry.lastLogin < 86400 %} 
                 <td class="{{tables.data_table_column_class (tableName, columns, 'lastLogin')}}" style="color: red !important"> {% if entry.lastLogin is not null%} {{entry.lastLogin | dates>
{% else %}
                 <td class="{{tables.data_table_column_class (tableName, columns, 'lastLogin')}}"> {% if entry.lastLogin is not null%} {{entry.lastLogin | date_full}} {% endif%} </td>
{% endif %}

Obviously that doesn't work and I think it's because entry.lastlogin is a variable in datetime… but… in this case which variable should be used to get the current date/time on the server?

or do you think there is a different approach?

Thank you for all your assistance

**Comment by kevinpapst** [comment, marked as answer]
It is simple, once you know that you can create a datetime by using the twig function `create_date('now')`.
```twig
{% if entry.lastLogin is not null and entry.lastLogin < create_date('-86400 seconds') %}
   ...
{% endif %}
```

**Reply by TecnicoDelphi** [reply]
To quote c&c:
mission accomplished :-P
thank you!!

**Reply by kevinpapst** [reply]
Sorry, there was a logical problem, I edited the answer. But you probably found that already.
